lib/upgrade_format_task.rb

```diff
@@ -44,7 +44,9 @@ def upgrade(store, name, options = {})
 
     if !errors.empty?
       Machinery.logger.error errors.join("\n")
-      raise Machinery::Errors::UpgradeFailed.new(errors.join("\n"))
+      exception = Machinery::Errors::UpgradeFailed.new(errors.join("\n"))
+      exception.hint_proc = -> { Hint.upgrade_format_force(name: name || "--all") }
```

Comment:
Looks good to me as well. It creates a slight asymmetry, though. To be consequent, we should then express printing a hint as something like

```ruby
Hint.print(:upgrade_format_force, name: name)
```